Include/internal/pycore_magic_number.h

```diff
@@ -261,6 +261,7 @@ Known values:
     Python 3.14a1 3606 (Specialize CALL_KW)
     Python 3.14a1 3607 (Add pseudo instructions JUMP_IF_TRUE/FALSE)
     Python 3.14a1 3608 (Add support for slices)
+    Python 3.14a1 3609 (Break up RETURN_VALUE)
```

Comment:
Alpha 1 came out a week ago, so I think this should be a2 instead:
```suggestion
    Python 3.14a2 3609 (Break up RETURN_VALUE)
```